standalone/src/DelphesHepMCReader.h

```diff
@@ -11,7 +11,7 @@
 #include "classes/DelphesClasses.h"
 #include "classes/DelphesFactory.h"
 #include "classes/DelphesStream.h"
-#include "classes/DelphesHepMCReader.h"
+#include "classes/DelphesHepMCReader2.h"
```

Comment:
The `2` seems to be at the wrong place here, it should be the following:
```suggestion
#include "classes/DelphesHepMC2Reader.h"
```